caas/kubernetes/provider/bootstrap.go

```diff
@@ -1315,6 +1321,44 @@ func (c *controllerStack) controllerContainers(jujudCmd, controllerImage string)
 				},
 			},
 		}},
+		//Env: []core.EnvVar{{
```

Comment:
Please remove these before landing.

Reply:
So we don't need any of:
- env variables
- liveness probe
- readiness probe
- security context
?